### DISCUSSION TITLE
How to display the prefix "System" when selecting the Gherkin steps.

### DISCUSSION THREAD

**Discussion by t2hv33** [reqnroll/Reqnroll#84] [Ideas]
**Background**:
We have 7 systems that need testing.
Our Gherkin steps in each scenario average **30**.
We have more than **1300 Gherkin steps**.

How can we quickly select or determine which system the Gherkin steps belong to?

For example, we have assigned code names for systems:
- ECOMMERCE: SYS01
- GATEWAY: SYS02
- DEALER: SYS03
- FINTECH: SYS04
I don't want to directly modify the Gherkin steps; it's better to have the prefix automatically loaded or something similar.
```gherkin
Given [SYS01] the user is on the homepage
When [SYS02] the user submits the order
Then [SYS03] the dealer receives a confirmation email
```
Expectation: Have some color in the prompt of the ReqnRoll Extension in Visual Studio or a prefix indicating the system.
Do you guys have any suggestions?

**Comment by gasparnagy** [comment]
@t2hv33 Interesting... 
How do you structure the step definitions currently (multiple projects, folders, classes)?
